Resolves Issue/PR templates (#706)
DefangLabs/defang: .github/PULL_REQUEST_TEMPLATE.md

> 

Reviewer: ```suggestion
```

## Description

<!-- Concise description of what this PR is tackling. -->

## Linked Issues

<!-- See https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue -->

## Checklist

- [ ] I have performed a self-review of my code
- [ ] I have added appropriate tests

> - [ ] I have updated the Defang CLI help-text to reflect my changes, if necessary

Reviewer: ```suggestion
- [ ] I have updated the Defang CLI docs and/or README to reflect my changes, if necessary
```

## Description

- Concise description of what this PR is tackling.

## Linked Issues

This section will automatically populate with linked issues.

[To link an issue to this pull request, use one of the following keywords linked here in the pull request description](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword):

- `close`
- `closes`
- `closed`
- `fix`
- `fixes`
- `fixed`
- `resolve`
- `resolves`
- `resolved`

For example:

- `Closes #10`
- `Fixes octo-org/octo-repo#100`
- `Resolves #10, resolves #123, resolves octo-org/octo-repo#100`

### Screenshots (optional)

[Upload optional screenshots]

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Extended the README / documentation, if necessary
- [ ] My changes generate no new warnings
- [ ] Is the code readable? Prefer to see more self-explanatory and consistency
- [ ] Is the code tested properly? i.e. where appropriate, we're testing well-explained failures, behaviours, fixes, etc

> ##### Memes Are welcomed

Author: Good callout here! I agree with your line of thinking. I like the simpler approach for this too
